Library/Homebrew/cmd/audit.rb

```diff
@@ -121,10 +121,15 @@ def include?(s)
     @text.include? s
   end
 
-  def line_number(regex)
-    index = @lines.index { |line| line =~ regex }
+  def line_number(regex, skip = 0)
+    index = @lines.drop(skip).index { |line| line =~ regex }
     index ? index + 1 : nil
   end
+
+  def reverse_line_number(regex)
+    index = @lines.reverse.index { |line| line =~ regex }
+    index ? @lines.count - index : nil
```

Comment:
The following should also do the trick and avoids building and immediately discarding a new array:

```ruby
index = @lines.rindex { |line| line =~ regex }
index ? index + 1 : nil
```